rotkehlchen/db/history_events.py

```diff
@@ -752,17 +747,82 @@ def get_value_stats(
                 amount = deserialize_fval(
                     value=row[1],
                     name='total amount in history events stats',
-                    location='get_value_stats',
+                    location='get_amount_stats',
                 )
-                sum_of_usd_values = deserialize_fval(
-                    value=row[2],
-                    name='total usd value in history events stats',
-                    location='get_value_stats',
+                assets_amounts.append((asset, amount))
+            except DeserializationError as e:
+                log.debug(f'Failed to deserialize amount {row[1]}. {e!s}')
+        return assets_amounts
+
+    def get_amount_and_value_stats(
+            self,
+            cursor: 'DBCursor',
+            query_filters: str,
+            bindings: list[Any],
+            counterparty: str,
+    ) -> tuple[list[tuple[str, FVal, FVal]], FVal]:
+        """Returns the sum of the amounts received by asset and the sum of USD value
+        at the time of the events and the total USD value of all the assets queried.
```

Comment:
why USD value and not user's selected currency? Since now we can do the actual currency and it ends up taking the same amount of effort.